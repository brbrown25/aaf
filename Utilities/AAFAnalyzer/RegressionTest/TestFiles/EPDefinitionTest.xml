<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE aaf-file SYSTEM "EPDerivationChain.dtd">

<aaf-file>
   
	<!-- Test Operation Definitions -->
    <top-level name="Valid OperationGroup/OperationDefinition">
        <timeline-mob-slot edit-rate-denominator="1" edit-rate-numerator="1">
            <mono-audio-dissolve track-type="audio"/>
        </timeline-mob-slot>
    </top-level>
    
    <top-level name="Invalid OperationGroup/OperationDefinition">
        <timeline-mob-slot edit-rate-denominator="1" edit-rate-numerator="1">
			<operation-group operation-definition="chroma-key" track-type="audio"/>
        </timeline-mob-slot>
    </top-level>
    
</aaf-file>