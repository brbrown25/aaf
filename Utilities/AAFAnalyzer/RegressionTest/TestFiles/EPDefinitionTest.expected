        Name:   Edit Protocol Definition Test
        Desc:   Verify the use of DataDefinition objects throught the file.
        Result: Failed
        Reason: 
        Covered Requirements: REQ_EP_163
        Noted Requirements: None
        Passing Requirements: None
        Warning Requirements: None
        Failing Requirements: REQ_EP_162

            Name:   OperationDefinition Data Definition Value
            Desc:   An OperationDefinition object in an AAF file shall have a data definition value consistent with the data definition of an OperationGroup object that references it.
            Result: Failed
            Reason: OperationGroup in Mob slot with ID = 1 of Top-Level Composition "Invalid OperationGroup/OperationDefinition" has data definition value "Sound" and references OperationDefinition "Chroma Key" with data definition value "Picture".
            Covered Requirements: None
            Noted Requirements: None
            Passing Requirements: None
            Warning Requirements: None
            Failing Requirements: REQ_EP_162

